apisix/control/v1.lua

```diff
@@ -155,6 +234,13 @@ function _M.get_health_checker()
     if not info then
         return 404, {error_msg = err}
     end
+
+    local out = try_render_html({stats={info}})
```

Comment:
The return of `iter_and_find_healthcheck_info` is not correct.
The following code should be added
```
            if checks.active and checks.active.type then
                info.type = checks.active.type
            elseif checks.passive and checks.passive.type then
                info.type = checks.passive.type
            end
```